engine/inc/renderer/3d/bbox/bbox.hpp

```diff
@@ -41,9 +41,21 @@ class BBox : public CoreBBox {
   /** @returns the bottom face (nearer on y-axis) */
   const BBoxFace& getBottomFace() { return _bottomFace; }
 
+  /** @returns the lower (x, y, z) boundary of the box. */
+  const Vec4 min();

```

Comment:
Why `const`? Lets give programmer full choice what he want to do with it.
(the same for `max()`)

Reply:
Right!